Zend/zend_API.c

```diff
@@ -1413,6 +1413,33 @@ static zend_result update_property(zval *val, zend_property_info *prop_info) {
 	return zval_update_constant_ex(val, prop_info->ce);
 }
 
+zend_result zend_update_class_constant(zend_class_constant *c, zval *value, zend_class_entry *scope)
```

Comment:
Yes, right! I only passed it because there was one case (`ret = zend_update_class_constant(c, ret_constant, c->ce);` in `zend_constants.c`) where I *believed* that `ret_constant` is not `c->value`, but I've just double checked it and turned out that I was wrong. I'm happy to get rid of the parameter - I didn't like it at all either.